### ISSUE TITLE
BIM(Building information modeling) implementation

### ISSUE THREAD

**Issue by j209** [hannobraun/fornjot#2459]
Hi
Any plan to implement [BIM](https://en.wikipedia.org/wiki/Building_information_modeling) and [openbim](https://www.buildingsmart.org/about/openbim/) implementation in this project?

**Comment by hannobraun** [comment]
No, no such plans. I'm busy with much more basic stuff, and have no connection to the building world.

This could happen as a third-party add-on though. If such an add-on existed, and ran into limitations in Fornjot, I'd be happy to support it by making or merging any reasonable changes to lift those limitations.

---

Converting this into a discussion, as that's a more appropriate venue for answering questions.
